gui/src/components/maintenance/cortx-about.vue

```diff
@@ -128,11 +237,16 @@ export default class Cortxaboutpage extends Vue {
     this.$data.versionDetails = res.data;
     this.$store.dispatch("systemConfig/hideLoader");
   }
-
   public async getApplianceDetails() {
     const res = await Api.getAll(apiRegister.appliance_info);
     this.$data.serialNumber = res.data[0].serial_number;
   }
+  public async getSSLDetails(){
+  const res = await Api.getAll(apiRegister.ssl_details);
+  this.$data.issuerDetails = res.data.cert_details.issuer;
```

Comment:
Please add null check for res and res.data

Reply:
done